rc.local.copyme

```diff
@@ -0,0 +1,3 @@
+cd /home/warg/competition-2024-auto-landing/  # Update to actual path
```

Comment:
Nit: In [airside system](https://github.com/UWARG/computer-vision-python/blob/main/rc.local.copyme) we also log the time the code was launched. Should we consider adding this. I think it would be super helpful in debugging if the script was launched successfully.